### DISCUSSION TITLE
Modifying Vanilla Loot Tables but only enabling one item or the other to drop

### DISCUSSION THREAD

**Discussion by DiscordantHarmony1** [FabricMC/fabric-api#2778] [Mod Dev Support] [answered]
Dear Fabric Team or any other helpful developers, I'm fairly new to modding and have been attempting to have a modded item I created to have same behavior as flint from gravel except its from obsidian, I've gotten it to have the chance to drop but the original loot table drop still drops also, I've gathered it has something to do with the 
FabricLootTableBuilder.modifyPools(java.util.function.Consumer) function but have no idea how to go about implementing it into my code and can't find any documentation about it anywhere

**Comment by Juuxel** [comment, marked as answer]
Yep, `modifyPools` is the correct one. You pass it a lambda where you can modify the loot pools via builders as if you were creating new ones.

See the testmod code: https://github.com/FabricMC/fabric/blob/c08c81e81075516a69dfaf95c4767c3261ae19f9/fabric-loot-api-v2/src/testmod/java/net/fabricmc/fabric/test/loot/LootTest.java#L92

**Reply by DiscordantHarmony1** [reply]
It took a little finagling to correctly implement into my spaghetti code but I finally got it,  Thank you very much!
